### PR TITLE
Enable tide for CNI

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Review by geeknoid — APPROVED** [review]

File: prow/config.yaml [hunk 1]
```diff
@@ -120,7 +120,16 @@ branch-protection:
             contexts:
             - deploy/netlify
         cni:
+          required_pull_request_reviews: *tide_withcodeowners_reviews
```
Comment (clarketm) [inline prow/config.yaml]:
```suggestion

```

Now that #1666 has merged , the `tide_withcodeowners_reviews` and `tide_withcodeowners_restrictions` anchors are no longer defined. You should be able to remove these lines and use org *defaults* (or override as needed).

File: prow/config.yaml [hunk 2]
```diff
@@ -120,7 +120,16 @@ branch-protection:
...
+          restrictions: *tide_withcodeowners_restrictions
```
Comment (clarketm) [inline prow/config.yaml]:
```suggestion

```

**Review by clarketm — CHANGES_REQUESTED** [review]

File: prow/config.yaml (hunk 1 above)
Comment (clarketm) [reply prow/config.yaml]: Otherwise, there may be unintentional conflicts or reversions.

**Comment by howardjohn** [comment]
Thanks @clarketm , should be fixed now

**Review by geeknoid — APPROVED** [review]

**Review by clarketm — APPROVED** [review]

**Pull request merged**